dev_guide/templates.adoc

```diff
@@ -98,10 +98,9 @@ $ `osc process -f _<filename.json>_ | osc create -f -`
 ****
 
 You can override any link:../dev_guide/templates.html#parameters[parameters]
-defined in the JSON file by adding the `-v` option and any desired parameters.
-Note that the usage of the parameters is not restricted to only environment
-variables. You can use the parameter reference in any text field inside the
-template items.
+defined in the JSON file by adding the `-v` option followed by a
+comma-separated list of the form _<name>_=_<value>_.
+A parameter reference may appear in any text field inside the template items.
```

Comment:
Just to be sure, we OK with dropping the specific "not restricted to only environment variables" bit? It was added in by @mfojtik's https://github.com/openshift/openshift-docs/pull/362 recently. The last sentence here does seem to still get that point across, just maybe not as overtly.